### DISCUSSION TITLE
Any plans for vite and sveltekit support?

### DISCUSSION THREAD

**Discussion by borsemayur2** [crewdevio/Snel#16] [Q&A]
It would be great to start sveltekit project (with vite) using snel. Is is possible?

**Comment by buttercubz** [comment]
sorry for the late response, now is not possible to support svelteKit and vite, are two project with a lot nodejs code and is not compatible with deno, but is there a draft for [deno adapter for svelteKit](https://github.com/sveltejs/kit/pull/1271) and [esbuild](https://deno.land/x/esbuild) now can be used in deno, with esbuild we can improvement compilation time in hot reloding to get similar experience like vite (with out [hmr](https://vitejs.dev/guide/features.html#hot-module-replacement))

**Comment by cg219** [comment]
Wonder if this will get attention now that there is support for npm packages in Deno
